pkg/store/sqlite/counterparty.go

```diff
@@ -337,6 +337,28 @@ func (s *Store) listContacts(tx *sql.Tx, counterparty *models.Counterparty) (err
 	return nil
 }
 
+// Returns a mapping of Contacts using their email as the keys; useful for comparing contacts we need to update/create.
+func (s *Store) listMapCounterpartyContactsByEmail(tx *sql.Tx, counterpartyID ulid.ULID) (contacts map[string]*models.Contact, err error) {
```

Comment:
I'm going to move this to `daybreak.go` since `Daybreak` is the only store interface that takes advantage of it; if other `store` methods were going to use it then we can move it back here.